feature/main/src/main/java/com/example/main/navigation/TopLevelDestination.kt

```
@@ -0,0 +1,14 @@
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import kotlin.reflect.KClass
+
+enum class TopLevelDestination(
+    val route : KClass<*>,
+    @DrawableRes val selectedIcon : Int,
+    @DrawableRes val unselectedIcon : Int,
+    val title : String
+) {
+
+}
```

detekt (1.23.7)

[warning] The class or object TopLevelDestination is empty.

(detekt.empty-blocks.EmptyClassBlock)